src/Explorer/Tabs/QueryTab/QueryTabComponent.test.tsx

```diff
@@ -16,6 +18,24 @@ import React from "react";
 
 jest.mock("Explorer/Controls/Editor/EditorReact");
 
+const loadState = (path: StorePath) => {
+  if (
+    path.componentName === AppStateComponentNames.QueryCopilot &&
+    path.subComponentName === CopilotSubComponentNames.toggleStatus
+  ) {
+    return true;
+  } else {
+    return undefined;
+  }
+};
+
+jest.mock("Shared/AppStatePersistenceUtility", () => ({
+  loadState,
+  AppStateComponentNames: {
+    QueryCopilot: "QueryCopilot",
+  },
+}));
```

Comment:
I don't think you need to do it now, but I'm noticing a pattern for testing state loading/saving that might be worth refactoring into a helper 🤔

Reply:
Turns out we don't need to mock `loadState`, we can use the real api to persist the state with the right value before the test (and cleanup afterwards). So I replaced this code with the actual call to set the state in the tests.